src/helpers/assertStringsWithinText.ts

```diff
@@ -0,0 +1,28 @@
+import { ReactTestRendererNode } from 'react-test-renderer';
+
+export const assertStringsWithinText = (
+  rendererJSON: ReactTestRendererNode | null | ReactTestRendererNode[]
+) => {
+  if (!rendererJSON) return;
+
+  if (Array.isArray(rendererJSON)) {
+    rendererJSON.forEach(assertStringsWithinText);
+    return;
+  }
+
+  if (typeof rendererJSON === 'string') {
+    return;
+  }
+
+  if (rendererJSON.type !== 'Text') {
```

Comment:
Looks like having `<Text>` disables any descending down in the tree search. Does RN actually supports something like `<Text><View>AAA</View></Text>`? Is putting `<View> ` inside `<Text>` allowed in RN runtime?

Reply:
Oh I thought it was allowed but it actually isn't, good catch ! I'll fix it and add a test case for it

Reply:
fixed